node/src/test/scala/coop/rchain/node/RhoExprToParSpec.scala

```diff
@@ -0,0 +1,71 @@
+package coop.rchain.node
+
+import cats.syntax.all._
+import coop.rchain.models.syntax._
+import coop.rchain.node.api.WebApi._
+import org.scalacheck.Arbitrary._
+import org.scalacheck.{Arbitrary, Gen}
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.propspec.AnyPropSpec
+import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks
+
+class RhoExprToParSpec extends AnyPropSpec with ScalaCheckDrivenPropertyChecks with Matchers {
+  property("WebAPI RhoExpr <-> Par conversion work correct") {
+    forAll { rhoExpr: RhoExpr =>
+      // TODO: Remove after finishing debugging
+      // println(expected)
+      // println(s"  $actual")
+
+      exprFromParProto(rhoExprToParProto(rhoExpr)) shouldBe rhoExpr.some
```

Comment:
You can even define this function `RhoExpr => RhoExpr` with function composition.
```scala
val rhoExprToRhoExpr = rhoExprToParProto >>> exprFromParProto
// or
val rhoExprToRhoExpr = rhoExprToParProto andThen exprFromParProto
```